ISISComputingGroup/ibex_gui
# Ticket 4785: Added dummy focusing button

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/4785

### To test

* Start up a mk3 chopper (in recsim will do) - probably best to do this on DEMO as requires DAQmx
* In mk3 chopper OPI change the phase
* Confirm no visible controls now have focus

---

#### Code Review


### Final Steps
